scripts/deploy-resource-quota.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# See documentation for more information on limit ranges:
+# https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+
+oc apply -f ./test-target/resource-quota.yaml --namespace "$TNF_EXAMPLE_CNF_NAMESPACE" 
```

Comment:
Use descriptive flags in scripts: --filename instead of -f

Reply:
Newline added and descriptive flag used.